usecases/MinimumWorkingExample/upload_scripts/doit_upload.py

```diff
@@ -1,29 +1,29 @@
 import logging
 import os
 import sys
+import re
 from collections import defaultdict
 from math import isnan
 from pathlib import Path
 from pprint import pformat
 from typing import Union
 
-import pandas as pd
 import yaml
 from dateutil.parser import parse
 from pybis.sample import Sample
-from datetime import datetime
 from lebedigital.openbis.interbis import Interbis
-
+from typing import List, Dict
```

Comment:
Which python version is used in LebeDigital? From 3.10 (I think) on you don't need to import the types anymore

Reply:
I thinks its 3.9 currently. This might (hopefully) change when we upgrade to fenicsxconcrete.